llvm/include/llvm/Analysis/TargetTransformInfo.h

```diff
@@ -622,6 +622,9 @@ class TargetTransformInfo {
     /// Don't allow runtime unrolling if expanding the trip count takes more
     /// than SCEVExpansionBudget.
     unsigned SCEVExpansionBudget;
+    /// Allow runtime unrolling multi-exit loops. Should only be set if the
```

Comment:
Add something to indicate that this doesn't disable the detection of profitable cases in generic code?

Reply:
done thanks